settings/com.endlessm.settings.gschema.override.in

```diff
@@ -39,7 +39,7 @@ button-layout=':minimize,maximize,close'
 
 # Specify the default background image
 [org.gnome.desktop.background]
-picture-uri='eos:///default'
+picture-uri='file:///usr/share/eos-media/desktop-background-C.jpg'
```

Comment:
If this is the only file from eos-media that is truly required as part of the baseline OS anymore, would it be better to move it into eos-theme so that things like the installer could drop eos-media?